sequoia/settings/offline_rl/setting.py

```diff
@@ -0,0 +1,110 @@
+import d3rlpy
+import gym
+from gym.wrappers import RecordEpisodeStatistics
+from matplotlib import pyplot as plt
+from sklearn.model_selection import train_test_split
+from typing import ClassVar, List, Tuple, Dict, Any
+
+from sequoia import Results
+from sequoia.settings.base import Setting
+from torch.utils.data import DataLoader
+from sequoia.settings.base import Method
+from dataclasses import dataclass
+
+from simple_parsing.helpers import choice
+
+
+
+@dataclass
+class OfflineRLResults(Results):
+
+    # TODO: Write these methods
+    def summary(self) -> str:
+        return f"Offline RL results: {self.objective_name} = {self.objective}"
+
+    def make_plots(self) -> Dict[str, plt.Figure]:
+        return {}
+
+    def to_log_dict(self, verbose: bool = False) -> Dict[str, Any]:
+        return {}
+
+    # Metrics from online testing
+    test_rewards: list
+    test_episode_length: list
+    test_episode_count: list
+
+    objective_name: ClassVar[str] = "Average Reward"
+
+    @property
+    def objective(self):
+        return sum(self.test_rewards) / len(self.test_rewards)
+
+
+# TODO: smarter way to do this, like a dict { 'offline_datasets_from_d3rlpy': set() } ?
+offline_datasets_from_d3rlpy = {'cartpole-replay', 'cartpole-random'}
```

Comment:
Are we only adding cartpole? Can't we add more of their datasets?

Reply:
added mujoco, atari